metriq_gym/process.py

```diff
@@ -46,16 +47,17 @@ def get_job_result_quantinuum(job, partial_result: BenchJobResult):
     return partial_result
 
 
-def poll_job_results(jobs_file: str, job_id: str) -> list[BenchJobResult]:
```

Comment:
Are those changes to the polling code intentional, or the outcome of a broken merge?

Reply:
Yes, these changes are intentional. As we are only providing the `job_id` now, we can remove the. `jobs_file` from the argument.